xbmc/pvr/epg/EpgInfoTag.h

```diff
@@ -280,6 +280,12 @@ namespace PVR
     int SeriesNumber(void) const;
 
     /*!
+     * @brief The series link for this event.
+     * @return The series link or empty string, if not available.
+     */
+    std::string SeriesLink() const;
```

Comment:
I just followed the style of the other getters for this class. Wanted to have it consistent at least.